bin/plotting/pycbc_plot_bank_corner

```diff
@@ -0,0 +1,282 @@
+#!/usr/bin/python
+# Copyright (C) 2023 Gareth S. Cabourn Davies
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+import sys
+import h5py
+import numpy as np
+import argparse
+import logging
+from pycbc import init_logging
+import pycbc.version
+from pycbc import __version__
+from pycbc.results.plot import (add_style_opt_to_parser, set_style_from_cli)
+from pycbc.io import FieldArray
+from pycbc.inference import option_utils
+from pycbc.tmpltbank import bank_conversions as bconv
+from pycbc.pnutils import get_imr_duration
+from pycbc.results.scatter_histograms import create_multidim_plot
+from pycbc.results import metadata
+
+conversion_options = bconv.conversion_options
+
+parser = argparse.ArgumentParser(usage='pycbc_plot_bank_corner [--options]',
+                    description='Plots various parameters against one '
+                          'another for pycbc banks in an hdf file.')
```

Comment:
There is a "feed two birds with one scone" case here: add a docstring to the entire file with this description, then just use `description=__doc__` when you instantiate the ArgumentParser.